cli/internal/cmd/verify_test.go

```diff
@@ -208,18 +208,20 @@ func TestVerify(t *testing.T) {
 
 func TestFormatDefault(t *testing.T) {
 	testCases := map[string]struct {
-		doc     string
+		doc     []byte
+		attCfg  config.AttestationCfg
 		wantErr bool
 	}{
 		"invalid doc": {
-			doc:     "invalid",
+			doc:     []byte("invalid"),
+			attCfg:  &config.AzureSEVSNP{},
```

Comment:
Do we anticipate adding more test cases? If not, I'd make this one case and remove the loop.

Reply:
There should be more cases for this test.
Currently it doesn't do anything helpful